bindings/wallet-cordova/tests/src/main.js

```diff
@@ -46,6 +46,7 @@ const symmetricCipherDecrypt = promisifyP(primitives.symmetricCipherDecrypt);
 const settingsGet = promisifyP(primitives.settingsGet);
 const settingsNew = promisifyP(primitives.settingsNew);
 const fragmentId = promisifyP(primitives.fragmentId);
+const ttlFromDate = promisifyP(primitives.ttlFromDAte);
```

Comment:
There's a capitalization typo, and the whole API should be renamed:

```suggestion
const blockDateFromSystemTime = promisifyP(primitives.blockDateFromSystemTime);
```